source/cloud/cloud-administration/site-configuration.rst

```diff
@@ -32,27 +32,27 @@ Mattermost in-product notices keep users and administrators informed of the newe
 Administrator Notices
 ^^^^^^^^^^^^^^^^^^^^^
 
-Administrator notices are used to inform System Admins when a new server version is available, or if a server upgrade is recommended due to ending support life cycles. System Admins may also receive notices about recommended server configuration options to optimize the user experience of their deployment.
+Administrator notices are used to inform System Admins of important updates or recommended configuration options to optimize the user experience of their deployment.
 
-Administrator notices can be disabled in the **System Console > Notices** page.
+Administrator notices can be disabled in the **System Console > Site Configuration > Notices** page.
 
 End User Notices
 ^^^^^^^^^^^^^^^^
 
-End user notices are used to inform users and Admins of new feature enhancements and when new desktop versions are available. They can be disabled in **System Console > Notices**.
+End user notices are used to inform users and Admins of new feature enhancements and when new desktop versions are available. They can be disabled in **System Console > Site Configuration > Notices**.
 
 FAQs
 ^^^^
 
 Are notices enabled by default?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in **System Console > Notices**.
+Notices are enabled by default for all Mattermost users. System Admins may choose to disable administrator or end user notices in **System Console > Site Configuration > Notices**.
 
-Will I still receive notices if my server is air-gapped?
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Will I still receive notices if my workspace is air-gapped?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-No, the Mattermost server requires a connection to the internet to receive notices.
+No, the Mattermost workspace requires a connection to the internet to receive notices.
```

Comment:
This question I believe is not relevant for cloud, there is no possibility for a cloud workspace to be air-gapped.

Reply:
Thanks, Maria! I'll remove this Q&A.